internal/services/springcloud/spring_cloud_service_resource.go

```diff
@@ -87,6 +87,13 @@ func resourceSpringCloudService() *pluginsdk.Resource {
 				}, false),
 			},
 
+			"default_build_service_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  true,
+				ForceNew: true,
+			},
```

Comment:
can we remove this feature flag? this should probably be created by default

Reply:
Sure, I'll remove this flag in this PR.